experimental/sql/src/ibis_frontend.py

```diff
@@ -105,11 +105,10 @@ def visit(  #type: ignore
 
 @dispatch(ibis.expr.operations.relations.InnerJoin)
 def visit(op: ibis.expr.operations.relations.InnerJoin) -> Operation:
-  return id.InnerJoin.get(
-      Region.from_operation_list([visit(op.left)]),
-      Region.from_operation_list([visit(op.right)]),
-      Region.from_operation_list(
-          [visit(op.predicates[0])] if op.predicates else []))
+  if op.predicates:
+    raise Exception(f"theta joins can not yet be lowered")
```

Comment:
Can you not lower the `InnerJoin` to *two* ops, a `CartesianProduct` followed by a `Filter`?